.github/workflows/fields.yaml

```diff
@@ -0,0 +1,22 @@
+name: Build Fields
+on:
+  push:
```

Comment:
Had a conversation with @dwsilk - he thought that the script could be triggered on push to your branch and also on push to master, looks like you can use `branches-ignore` to exclude `master`.
[https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#onpushpull_requestbranchestags](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#onpushpull_requestbranchestags)